infrastructure/resource_group_level/modules/webapps/external-api.bicep

```diff
@@ -88,6 +88,7 @@ module externalApiWebapp 'webapp.bicep' = {
     privateEndpointsSubnetId: privateEndpointsSubnetId
     resourceNameFragment: resourceNameFragment
     scmMinTlsVersion: '1.2'
+    selfHostnameEnvironmentVariable: 'EXTERNAL_API_URL'
```

Comment:
`external-api` needs to know its own address for emails. We can actually do that fairly easily as we set up the application settings in a separate resource from the webapp.